cobalt/network/cookie_jar_impl.cc

```diff
@@ -63,6 +65,9 @@ CookieJarImpl::CookieJarImpl(net::CookieStore* cookie_store,
     : cookie_store_(cookie_store), network_task_runner_(network_task_runner) {
   DCHECK(cookie_store_);
   DCHECK(network_task_runner);
+
+  const char* result = inet_ntop(0, nullptr, nullptr, 0);
```

Comment:
I assume this is just for testing the change.

Reply:
Correct, I'll remove before submitting